lib/matplotlib/tests/test_simplification.py

```diff
@@ -13,11 +15,14 @@
 from matplotlib.path import Path
 
 
-# NOTE: All of these tests assume that path.simplify is set to True
-# (the default)
+@pytest.fixture
+def set_simplify_and_threshold():
```

Comment:
Hmm. In fact, using `scope='module'` makes some tests fail. Possibly b/c it's only run once in this module, meaning the test that turns off simplification would effect later tests? Don't know for sure. I'll leave it as is for now.